R-package/configure.ac

```diff
@@ -28,8 +28,15 @@ fi
 
 if test `uname -s` = "Darwin"
 then
-  OPENMP_CXXFLAGS='-Xclang -fopenmp'
-  OPENMP_LIB='-lomp'
+  if command -v brew &> /dev/null
+  then
+    HOMEBREW_LIBOMP_PREFIX=`brew --prefix libomp`
```

Comment:
Actually, can we even assume homebrew is available? I don't feel comfortable around changes in the R package as CRAN has one of the strictest and most specialized environment requirements.

Reply:
NVM, I notice the condition now.